### PR TITLE
#164069212 Set up and deploy Base Application to Heroku

### PR DESCRIPTION
**Pull request by wasibani-roy**
**What does this PR do**
Setup and Deploy the base application to Heroku

**Description of tasks to be completed**
- Ensure that an application has been created on Heroku that I will link to our project
- Add a procfile to the repo

**How should this be manually tested??**
This can be tested by going to our staging application [link](https://ah-backend-aquaman-staging.herokuapp.com/api/user) and ensuring that the app loads fine

**Any background context you want to provide**

- N/A

**What are the relevant pivotal tracker stories**
[#164069212](https://www.pivotaltracker.com/story/show/164069212)

### CODE REVIEW

**Review by CryceTruly — COMMENTED** [review]
Hey @wasibani-roy ,As requested by @marthamareal , please consider hiding secret information, as well as cleaning up the requirements file to include only the current project dependencies.

**Comment by joelethan** [comment]
Thank you @wasibani-roy, Good job ... I have tested it and it is working fine.

**Comment by Peace-Apple** [comment]
I have checked the deployment and it is fine, good work.

**Review by BagzieGracious — DISMISSED** [review]
@wasibani-roy, I have tested your deployment and its working. Good work

**Review by Peace-Apple — DISMISSED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -25,3 +25,8 @@ six==1.12.0
 typed-ast==1.3.1
 urllib3==1.24.1
 wrapt==1.11.1
+gunicorn==19.9.0
+psycopg2==2.7.6.1
+psycopg2-binary==2.7.6.1
+PyJWT==1.6.4
```
Comment (marthamareal) [inline requirements.txt]: Good work @wasibani-roy , however `PyJWT` is already added on line 16

Comment (wasibani-roy) [reply requirements.txt]: Thank you @marthamareal I have redone the rebase it should be corrected now

**Review by joelethan — DISMISSED** [review]

**Review by BagzieGracious — DISMISSED** [review]

**Review by Peace-Apple — DISMISSED** [review]
LGTM

**Review by Peace-Apple — DISMISSED** [review]

**Review by BagzieGracious — DISMISSED** [review]

**Review by joelethan — DISMISSED** [review]

**Comment by CryceTruly** [comment]
LGTM

**Review by Peace-Apple — DISMISSED** [review]
LGTM

**Review by joelethan — APPROVED** [review]

**Review by BagzieGracious — APPROVED** [review]
LGTM

**Review by Peace-Apple — APPROVED** [review]
LGTM

**Pull request merged**